### PR TITLE
Mechanism OAUTHBEARER - Patch issue 591

### PR DESCRIPTION
**Pull request by ericmagna**
Modify 2 files to make an connection in mechanism oauthbearer.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -334,6 +335,33 @@ akhq:
...
+
+#### Requirement Library Strimzi
+
+> The kafka brokers must be configured with the Strimzi library and an OAuth2 provider (Keycloak example).
+
+> This ![repository](https://github.com/strimzi/strimzi-kafka-oauth) contains documentation and examples.
+
+#### Configuration Bootstrap
+
+> It is necessary to compile AKHQ to integrate the Strimzi libraries.
```
Comment (tchiotludo) [inline README.md:348]: It's NOT 😅

**Review by tchiotludo — CHANGES_REQUESTED** [review]
Just a typo on readme 😅

File: README.md (hunk 1 above)
Comment (ericmagna) [reply README.md]: it's corrected.

**Comment by tchiotludo** [comment]
Thanks :)

**Comment by kc89rst** [comment]
> Thanks :)
Do you have an estimate on when it will be available on Dockerhub (https://hub.docker.com/r/tchiotludo/akhq/tags)?

**Comment by tchiotludo** [comment]
It's available on `dev` tag right now ;)

**Comment by kc89rst** [comment]
> It's available on `dev` tag right now ;)

Ultra cool, hope my boss will not kill me for using a dev tag :P

**Comment by tchiotludo** [comment]
Don't say him ;) 
Because to be honest, I don't plan a new release soon for now ;)

**Pull request merged**